elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -34,3 +34,53 @@ window.PLRTE = function (uuid, options) {
     );
   });
 };
+
+// Override default implementation of 'formula'
+
+var Embed = Quill.imports.parchment.Embed;
+
+class MathFormula extends Embed {
+  static create(value) {
+    const node = super.create(value);
+    if (typeof value === 'string') {
+      this.updateNode(node, value);
+    }
+    return node;
+  }
+
+  static updateNode(node, value) {
+    // Wait for the page to be fully loaded. Mathjax will not work unless the page is fully loaded.
+    if (document.readyState === 'complete') {
+      let html = MathJax.tex2chtml(value);
+      let formatted = html.innerHTML;
+      // Without trailing whitespace, cursor will not appear at end of text if LaTeX is at end
+      node.innerHTML = formatted + '&#8201;';
+      MathJax.typeset();
+      node.contentEditable = 'false';
+      node.setAttribute('data-value', value);
+    } else {
+      var n = node;
+      window.addEventListener(
+        'load',
+        function () {
+          MathFormula.updateNode(n, value);
+        },
+        false
+      );
+    }
+  }
+
+  static value(domNode) {
+    return domNode.getAttribute('data-value');
+  }
+
+  html() {
+    const { formula } = this.value();
+    return `<span>${formula}</span>`;
+  }
```

Comment:
Taking another look at this, I'm not sure how this is supposed to work - `this.value()` expects a DOM node as an argument and doesn't return anything that we can destructure `formula` out of. I can't find a reference to this function in the Quill docs. Do you know how we can test this?

Reply:
[I copied it directly out of formula.js just in case it was used](https://github.com/quilljs/quill/blob/9cf0285caa6514356a7bba9db132ec7229eb254c/formats/formula.js#L23), though you do make a good point that it doesn't seem to be used, or usable?

Reply:
Ah, it looks like that original one is broken too. I'd rather we just drop this - AFAICT, if it _were_ used, it would break.